PokeAlarm/WebhookStructs.py

```diff
@@ -71,7 +75,8 @@ def pokemon(data):
             'gender': get_pokemon_gender(check_for_none(int, data.get('gender'), '?')),
```

Comment:
You could set gender to:
`'gender': get_pokemon_gender(check_for_none(int, data.get('gender'), get_form_name(int(form_raw)))),`
And remove your changes in manager.py to have form replace gender.